main.go

```diff
@@ -0,0 +1,45 @@
+/*
+ *Author: Stefan
+ *Date: 11/16/2019
+ *Last changes: 11/17/2019
+ *Task: Implement the following methods: End - defines the end of the shape, Perimiter, Area
+**/
+
+package main
+
+import "fmt"
+
+//Point structure
+type Point struct {
+	x, y int
+}
+
+//Square structure
+type Square struct {
+	start Point
+	a     uint
+}
+
+//End method
+func (p *Square) End() (int, int) {
```

Comment:
This function should return `Point` structure.

Reply:
Great, I fixed it! Thanks a lot.